### ISSUE TITLE
Launch updated process and announce to teams

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#94452]
## User Story
As a Governance team member, I want to launch the new Platform Orientation process on Platform Website so that VFS teams can begin engaging in the new process.

Assignee: @KKitagawa-Bosch 
Peer Reviewer: @allison0034 @shiragoodman 

## Description
Governance team has done all the work associated with splitting onboarding from orientation (AKA splitting educational tasks from the administrative tasks). We have streamlined the New VFS Team Member gh ticket and made it much easier for VFS teams to access  educational materials.

The purpose of this ticket is to launch the new process and announce the changes to VFS teams. 

### Implementation notes

- It looks like, when we are ready to go live, we will need to remove the orientation-epic.md file entirely and replace it with a new new-vfs-team-member.yml file using the code that you used in your personal repo. Then we would need to update the link in the [guidance for new team members page](https://depo-platform-documentation.scrollhelp.site/getting-started/guidance-for-new-teams-and-team-members) on platform website to link to the new issue template URL. That work is not part of this ticket, though - just a note for the future.
- Add labels: ["platform-orientation", "new-vfs-team-member", "platform-tech-team-support"]
- Reevaluate the "educational tasks" check box once that page is complete

### Live pages
- [New YAML Template](https://github.com/department-of-veterans-affairs/va.gov-team/blob/017c82de649040fea38e37e142ef7e3413e18ec1/.github/ISSUE_TEMPLATE/New-VFS-Team-Member.yml)
- [Platform Orientation](https://depo-platform-documentation.scrollhelp.site/getting-started/platform-orientation)
- [Guidance for New Teams ](https://depo-platform-documentation.scrollhelp.site/getting-started/guidance-for-new-teams-and-team-members)
- [Educational tasks](https://depo-platform-documentation.scrollhelp.site/getting-started/educational-tasks)
- [Orientation Sessions](https://depo-platform-documentation.scrollhelp.site/getting-started/orientation-sessions)

### Drafted pages
- 

## Tasks (in no particular order, unless specified)
- [x] Publish all new Platform Website pages and changes to existing Platform Website pages
- [x] Ensure the new yaml template for the New VFS Team Member template is published and the old md template has been archived. See Implementation notes above for more info).
- [x] Validate that the link to the new yaml template is included on the appropriate Platform Orientation page
- [x] Write a change announcement following the [Slack change announcement format](https://vfs.atlassian.net/wiki/spaces/PMCP/pages/2290352129/Slack+Change+Announcement+format)
- [x] Once change announcement has been peer reviewed, all pages are live and all old templates have been archived, post the change announcement to the #vfs-change-announcements channel, and cross share to #vfs-platform-support, #vfs-all-teams, #platform-governance-team, and #platform-team (for the platform-team channel, tag Alyssa Galion and Brandon Dech)
- [x] Notify Shira so she can share in Team of Teams

## Peer Review
_To be completed by peer reviewer_
- [x] User story and acceptance criteria are met

## Acceptance Criteria
- [x] Governance has gone live with the new Platform Orientation process
- [x] The new Platform Orientation process has been announced to VFS teams
- [x] the new Platform Orientation process has been shared with Platform Support team (Alyssa and Brandon) so that they know what to expect moving forward.

## Team Notification
- [x] Slack post on #platform-governance-team
- [ ] Presentation at Weekly Initiative Meeting
- [ ] No notification needed

### Refinement
- [x] Ticket has user story, description, tasks, and acceptance criteria
- [x] Ticket has been sized 
- [x] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [x] Ticket has been discussed as a team at Planning
- [x] Ticket has been assigned to appropriate initiative or quarterly epic
- [x] Ticket has been assigned to a Sprint
- [x] Ticket has been moved to the Planned pipeline in the Governance team board
- [x] Ticket as an assignee and peer reviewer

**Comment by shiragoodman** [comment]
@KKitagawa-Bosch Is this work in progress? We're 3 days into the sprint and I would've expected this to have started. Please lmk.

**Comment by KKitagawa-Bosch** [comment]
@shiragoodman discussed at standup yesterday but for posterity: yes this has started! Not as far into it as I would have liked because collab cycle stuff late last week was heftier than usual, but I expect to be pretty much done by the end of today!

**Comment by KKitagawa-Bosch** [comment]
@allison0034 [Slack Change Announcement](https://vfs.atlassian.net/wiki/spaces/PPST/pages/3662544897/Platform+Orientation+Change+Announcement) is drafted and ready for review. Shira said she would prefer to wait until after the holiday for this to go live, so I marked the date as Monday.

**Comment by allison0034** [comment]
@KKitagawa-Bosch have you published any of the changes? Also we need to get the YAML file in the right repo. I dont think you want to post the change announcement until everything is published right?

**Comment by KKitagawa-Bosch** [comment]
@allison0034 everything is updated and links are in live docs! Once you've reviewed those, I'll post the change announcement!

**Comment by KKitagawa-Bosch** [comment]
All checkboxes are checked, so closing this!
